src/hotspot/share/runtime/synchronizer.hpp

```diff
@@ -25,6 +25,7 @@
 #ifndef SHARE_RUNTIME_SYNCHRONIZER_HPP
 #define SHARE_RUNTIME_SYNCHRONIZER_HPP
 
+#include "logging/logStream.hpp"
```

Comment:
If you need to put MonitorList in the header file, use a forward declaration for LogStream instead of #including logstream.hpp.

Reply:
I can see if that will work.